src/Component/spec/Model/AbstractTranslationSpec.php

```diff
@@ -11,28 +11,48 @@
 
 declare(strict_types=1);
 
-namespace spec\Sylius\Component\Resource\Model;
+namespace spec\Sylius\Resource\Model;
```

Comment:
no change here, the legacy file has been moved and the new one has been placed here.